docs/guide/queries.md

```diff
@@ -1 +1,71 @@
 # Queries
+
+## Definition:
+
+Queries are dependencies to an async source of data. They allow us to declaratively fetch, cache and refresh the data.
+
+::: info
+Queries are meant to **read** data. In terms of REST, for example, queries would handle `GET` requests. If you need to **write** (or mutate) data, you can use [mutations](./mutations.md).
+:::
+
+## Defining a query:
+
+The API to define a query is the `useQuery` composable:
+
+```vue twoslash
+<script setup lang="ts">
+import { useQuery } from '@pinia/colada'
+
+const { data, error, isFetching } = useQuery({
+  key: ['todos'],
+  query: () => fetch('/api/todos').then((res) => res.json())
+})
+</script>
+
+<template>
+  <main>
+    <div v-if="isFetching">
+      Loading
+    </div>
+    <div v-else-if="error">
+      Oups, an error happened...
+    </div>
+    <div v-else>
+      <Todo v-for="todo in data" :key="todo.id" :todo />
+    </div>
+  </main>
+</template>
+```
+
+This composable:
+- accepts an option object, which configures the key of the query, how to fetch de data, and options related to the cache and its revalidation
+- returns the state of the query, which can be used in the UI, and some methods to declaratively revalidate the cache if needed
```

Comment:
Let's cover the basics first: the query, then the key, what is expected from them. Then explain how the query is automatically triggered when needed (no need to go into details yet) and that it will update `data`, and others enabling a declarative approach